app/src/main/res/values/dimens.xml

```diff
@@ -2,4 +2,7 @@
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
+    <dimen name="fab_margin">16dp</dimen>
+    <dimen name="contest_app_textsize">36sp</dimen>
```

Comment:
We should also make clearer titles for the dimensions, something like title_text_size and sub_title_text_size are more clear. having the app name in the textsize is probably unnecessary